src/ScenarioUtils/__test__/ScenarioUtils.spec.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import ScenarioUtils from '..';
+
+const scenarioList = [
+  { name: 'existing scenario' },
+  { name: 'existing scenario 2' },
+  { name: 'existing scenario 3' },
+  { name: 'existing scenario 4' },
+];
+
+describe('scenarioNameIsValid', () => {
+  test.each`
+    dataStr                   | expectedRes
+    ${'   new scenario'}      | ${'forbiddenCharsInScenarioName'}
+    ${'existing scenario'}    | ${'existingScenarioName'}
+    ${'new scenario *****'}   | ${'forbiddenCharsInScenarioName'}
+    ${'new +scenario'}        | ${'forbiddenCharsInScenarioName'}
+    ${'new ,scenario'}        | ${'forbiddenCharsInScenarioName'}
+    ${'existing scenario 2'}  | ${'existingScenarioName'}
+    ${'new.scenario'}         | ${null}
+    ${'new_scenario'}         | ${null}
+    ${'new & scenario'}       | ${'forbiddenCharsInScenarioName'}
+    ${'new % scenario *****'} | ${'forbiddenCharsInScenarioName'}
+    ${'new §scenario *****'}  | ${'forbiddenCharsInScenarioName'}
+    ${'new scenario'}         | ${null}
+    ${'new-scenario'}         | ${null}
+    ${'new---scenario'}       | ${null}
+    ${'existing scenario 3'}  | ${'existingScenarioName'}
+    ${'---new scenario'}      | ${'forbiddenCharsInScenarioName'}
+    ${'_new scenario'}        | ${'forbiddenCharsInScenarioName'}
+    ${'.new scenario'}        | ${'forbiddenCharsInScenarioName'}
+    ${'new*scenario'}         | ${'forbiddenCharsInScenarioName'}
+    ${'new=scenario'}         | ${'forbiddenCharsInScenarioName'}
+    ${'new+scenario'}         | ${'forbiddenCharsInScenarioName'}
+    ${'new scenario---'}      | ${null}
+    ${'existing scenario 4'}  | ${'existingScenarioName'}
+    ${'123 scenario'}         | ${null}
+    ${'   new scenario   '}   | ${'forbiddenCharsInScenarioName'}
+    ${'new scenario 132'}     | ${null}
```

Comment:
Could you add one more case to test trailing spaces in a scenario name conflicting with an existing scenario ? (e.g. "existing scenario    ")